ts/Dashboards/Components/Component.ts

```diff
@@ -730,6 +730,10 @@ abstract class Component {
             width,
             height
         });
+
+        fireEvent(this, 'redraw', {
+            component: this
+        });
```

Comment:
IDK if this is accurate. it is added in the `resize` method. Apart from that, there already is a `redraw` event in the Component Class. 
https://github.com/highcharts/highcharts/blob/4181cfcd97f85e7de70fb514333434ce1c49f7b0/ts/Dashboards/Components/Component.ts#L966

Reply:
Perhaps this should be changed?